src/main/java/haveric/recipeManager/recipes/CraftRecipeParser.java

```diff
@@ -88,7 +88,7 @@ public boolean parseRecipe(int directiveLine) throws Exception {
 
         recipe.setIngredients(ingredients); // done with ingredients, set 'em
 
-        if (recipe.hasFlag(FlagType.REMOVE)) {
+        if (recipe.hasFlag(FlagType.REMOVE) && !Version.has1_12Support()) {

```

Comment:
Ah, that makes sense. That kinda sucks as I'm sure everybody else who uses remove will have the same issues. It might be worth me spending a bit more time trying to solve the matching, but I'm just not sure of a good way at this time to do that.